common/notification/index.ts

```diff
@@ -241,3 +257,56 @@ async function deliverNotification(concreteNotification: ConcreteNotification, n
     }
 }
 
+/* When introducing new notifications, it might sometimes make sense to schedule them "in retrospective" once for all existing users,
+   as if the user took that action at actionDate */
+export async function actionTakenAt(actionDate: Date, user: Person, actionId: string, notificationContext: NotificationContext, apply: boolean) {
```

Comment:
Are we sure that we will never send mails to deactivated users?

Reply:
Very true, that check was missing. Good catch!

Reply:
Though at least in theory all matches of deactivated users should be dissolved (just noticed when trying to test this)